jch/html-pipeline
# Replace Test::Unit With Minitest

- Add minitest dependency to Gemfile test group
  - >= 4.2 since `ActiveSupport` depends upon minitest ~> 4.2 or ~> 5.1
- Define `HTML::Pipeline::Test` so, test suite can support `Minitest` 5 or 4
- Move `TestHelpers` into `HTML::Pipeline::Test`
  - Remove unused `assert_doesnt_include`
  - Rename `assert_includes` to `assert_include` since `Minitest` defines `assert_includes`
- Replace `Test::Unit` methods with matching `Minitest` methods
- Refactor `assert_nothing_raised` to use `assert_equal` since
  `Minitest` doesn't define `assert_nothing_raised`
  - See http://blog.zenspider.com/blog/2012/01/assert_nothing_tested.html
